Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2.yml

```diff
@@ -1709,7 +1807,5 @@ inputs:
   playbookInputQuery:
 outputs: []
 tests:
-- Test XDR Playbook
+- No tests (auto formatted)
 fromversion: 6.5.0
-marketplaces:
```

Comment:
you removed the marketplace label - is that intended?

Reply:
no, Im guessing this happened due to format command. done